### PR TITLE
WIP : retrieve item linked feature

### PR DESCRIPTION
**Pull request by stonebuzz**
Retrieve item linked feature

### CODE REVIEW

File: inc/object_item.class.php [hunk 1]
```diff
@@ -78,17 +136,209 @@ static function showItemsForSource(CommonDBTM $item) {
...
+         Ajax::updateItemOnSelectEvent("dropdown_$myname$rand", "results_$myname$rand",
+                                       $CFG_GLPI["root_doc"].
+                                          "/plugins/genericobject/ajax/dropdownTrackItemType.php",
+                                       $p);
+         echo "<span id='results_$myname$rand'>\n";
+         echo "</span>";
+
+         echo "</td><td class='center' width='30%'>";
+         echo "<input type='submit' name='add' value=\""._sx('button', 'Add')."\" class='submit'>";
+         echo "<input type='hidden' name='items_id' value='$instID'>";
```
Comment (btry) [inline inc/object_item.class.php]: suggestion : use Html::hidden(), HTML::input(),

File: ajax/dropdownTrackItemType.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+   $linked_data = $mainItemtypeLink->find(["itemtype" => $_POST["itemtype"]]);
+   $linked_id = [];
+   foreach ($linked_data as $key => $value) {
+      $linked_id[$key] = $value['items_id'];
+   }
+   $_POST['used'] = $linked_id;
+   Dropdown::show($_POST["itemtype"],$_POST);
+}else{
+   echo "";
+}
```
Comment (btry) [inline ajax/dropdownTrackItemType.php]:
```php

if (!isset($_POST["itemtype"]) || $_POST["itemtype"] == "0") {
   exit();
}

//load all  items_id already linked
$mainItemtypeLink = new $_POST['main_itemtype_link']();
$linked_data = $mainItemtypeLink->find(["itemtype" => $_POST["itemtype"]]);
$linked_id = [];
foreach ($linked_data as $key => $value) {
   $linked_id[$key] = $value['items_id'];
}
$_POST['used'] = $linked_id;
Dropdown::show($_POST["itemtype"],$_POST);
```

File: inc/object_item.class.php (hunk 1 above)
Comment (stonebuzz) [reply inc/object_item.class.php]: done

File: ajax/dropdownTrackItemType.php (hunk 2 above)
Comment (stonebuzz) [reply ajax/dropdownTrackItemType.php]: done

**Pull request closed without merging**